.github/workflows/release-edit-with-push.yml

```
@@ -37,4 +37,6 @@ jobs:
       # PRのnotesを更新
       - name: Update PR
         run: |
-          gh pr edit ${{ steps.get_pr.outputs.pr_number }} --body "${{ steps.changelog.outputs.changelog }}"
+          gh pr edit ${{ steps.get_pr.outputs.pr_number }} --body "$CHANGELOG"
+        env:
+          CHANGELOG: ${{ steps.changelog.outputs.changelog }}
```

ついでに念の為に pr_number も env 経由にしたいかもしれない

```suggestion
          gh pr edit "$PR_NUMBER" --body "$CHANGELOG"
        env:
          CHANGELOG: ${{ steps.changelog.outputs.changelog }}
          PR_NUMBER: ${{ steps.get_pr.outputs.pr_number }}
```